src/application/scene/Timer.java

```diff
@@ -1,14 +1,172 @@
 package application.scene;
 
+import javafx.animation.KeyFrame;
+import javafx.animation.Timeline;
+import javafx.event.ActionEvent;
 import javafx.fxml.FXML;
 import javafx.scene.control.Button;
+import javafx.scene.control.ToggleButton;
+import javafx.scene.control.ToggleGroup;
+import javafx.scene.text.Text;
+import javafx.util.Duration;
 
-public class Timer {
-	  @FXML private Button button_dashboard;
+public class Timer{
 
-	  /** Click handler for the dashboard button. */
-	  @FXML
-	  private void dashboard() {
-	    SceneManager.switchToDashboardScene();
-	  }
+	@FXML private ToggleButton pomodoroButton, shortBreakButton, longBreakButton;
+	@FXML private ToggleGroup timerButtons;
+	@FXML private Text countdownText;
+	@FXML private Button startButton;
+
+	private static final String POMODORO_TIME = "25:00";
+	private static final String SHORT_BREAK_TIME = "05:00";
+	private static final String LONG_BREAK_TIME = "15:00";
+	private static final String DISABLE_BUTTON_STYLE = "-fx-opacity: 1.0;";
+	private Time time = new Time();
+	private int numOfPomodoro = 0;
+	private static final int MAX_NUM_OF_POMODORO = 4;
+
+	private Timeline timeline = new Timeline(new KeyFrame(Duration.seconds(1), e -> {
+
+		// When one of the break timers finish, switch timer to prepare for a pomodoro
+		if (time.getMinutes()==0 && time.getSeconds()==0 && (shortBreakButton.isSelected() || longBreakButton.isSelected())){
```

Comment:
Could this be changed so that the pomodoro/break starts automatically when it switches over?

Reply:
When I looked at examples of pomodoro apps and webapps they usually have it so it lets the user start the new pomodoro/break